test/integration/content-picker/ContentPicker.e2e.cy.js

```diff
@@ -2,13 +2,8 @@ import localize from '../../support/i18n';
 
 // <reference types="Cypress" />
 const helpers = {
-    load(additionalProps = {}) {
-        cy.visit('/Elements/ContentPicker', {
-            onBeforeLoad: contentWindow => {
-                contentWindow.PROPS = additionalProps;
-            },
-        });
-
+    loadBasic(args = '') {
```

Comment:
how come we couldnt use `load` here?

Reply:
We can, however, at one point I had multiple load() functions for different URLs. for example, the first tests I updated used `elements-contentpicker-tests-e2e--with-pagination`, which I put in to the first load(). Then, the next set of tests used the `--basic` version, which is where I made `loadBasic`. Later, once I defined custom stories for this e2e test and minimized the need for passing args through the URL, I got rid of the "naked" `load()`.

loadBasic, however, was still a good candidate for passing args through the URL because the tests do legitimately use multiple values.

At the end, i felt that loadBasic was more specific than the simple load(), so I left it.

Reply:
I think you need to enhance the logic here if you are getting that issue such that the default is just the basic path and if they provide a new path you just pass it into the function and it will use that instead for that one tests

Reply:
Getting what issue? I don't think there's any difference between load() and loadBasic(), except in the name.

Reply:
ah, I think you mean, you'd like to see a wrapper function which takes a storybook "story ID" as an argument, but otherwise defaults to the basic story? 

if so, the reason i didn't end up making a shared function like that was because the same effect can be achieved with just straight cy.visitStorybook(), and to me, this approach is slightly more explicit about what we're loading.

to put it differently, the intent of loadBasic isn't to be a general function which takes in a story id **and** the query param args, but to be a specific function just for the basic case and any args it requires. there aren't other stories which require different args, so i didn't feel the need to make a more general function. I'd rather just keep the function specific to this one story id as loadBasic.